pkg/cache/nodes.go

```diff
@@ -234,9 +234,9 @@ func hasReadyCondition(node *v1.Node) bool {
 	return false
 }
 
-func triggerEvent(node *SchedulerNode, currentState string, eventType events.SchedulerNodeEventType) {
+func triggerEvent(node *SchedulerNode, currentState string, eventType SchedulerNodeEventType) {
 	log.Logger().Info("scheduler node event ", zap.String("name", node.name),
-		zap.String("current state ", currentState), zap.String("transition to ", string(eventType)))
+		zap.String("current state ", currentState), zap.String("transition to ", eventType.String()))
```

Comment:
Should probably use zap.Stringer() here rather than explicit .String() call.

Reply:
Done. Thanks.